openedx/features/course_experience/static/course_experience/js/CourseHome.js

```diff
@@ -4,13 +4,12 @@ export class CourseHome {  // eslint-disable-line import/prefer-default-export
   constructor(options) {
     // Logging for course tool click events
     const $courseToolLink = $(options.courseToolLink);
-    $courseToolLink.on('click', () => {
-      const courseToolName = document.querySelector('.course-tool-link').text.trim().toLowerCase();
+    $courseToolLink.on('click', (event) => {
+      const courseToolName = event.srcElement.text.trim().toLowerCase();
```

Comment:
From Robert's [comment](https://github.com/edx/edx-platform/pull/15630#discussion_r128832942)

Sorry I wasn't paying attention earlier, but we need a more stable id for these than the text (which can change and be translated). I'm not sure what we already have, and what's the write place to write it (data attribute?).
@andy-armstrong ^